Packs/ThreatConnect/Integrations/ThreatConnectV3/ThreatConnectV3.py

```diff
@@ -795,8 +799,21 @@ def tc_get_indicators_by_tag_command(client: Client, args: dict) -> None:  # pra
 
 def tc_get_indicator_command(client: Client, args: dict) -> None:  # pragma: no cover
     indicator = args.get('indicator')
-    response = tc_get_indicators_command(client, args, return_raw=True, indicator_id=indicator)  # type: ignore
+    indicator_id = ''
+    summary = ''
+    # We do this to check if the given indicator is an ID or a summary
+    try:
+        # If it's an int it means that it's an ID
+        int(indicator)  # type: ignore
+        indicator_id = indicator  # type: ignore
+    except ValueError:
+        # If not we'll treat it as a summary
+        summary = indicator  # type: ignore
+    response = tc_get_indicators_command(client, args, return_raw=True, indicator_id=indicator_id,
+                                         summary=summary)  # type: ignore
     ec, indicators = create_context(response, include_dbot_score=True)
+    if not indicators:
+        return_error(f'Could not find indicator: {indicator}')
```

Comment:
If you come up empty, you shouldn’t fail the command.

Reply:
Returned a message instead